src/objdump.py

```diff
@@ -89,7 +98,7 @@ class ArchSettings:
     "bgt-",
 }
 
-PPC_BRANCH_LIKELY_INSTRUCTIONS = {}
+PPC_BRANCH_LIKELY_INSTRUCTIONS: Set[str] = set({})
```

Comment:
```suggestion
PPC_BRANCH_LIKELY_INSTRUCTIONS: Set[str] = set()
```
to avoid constructing an empty dict and passing that to the set constructor